Microsoft.Identity.Web/NuGet.Config

```diff
@@ -0,0 +1,6 @@
+﻿<?xml version="1.0" encoding="utf-8"?>

```

Comment:
why do we need to add this file, @brentschmaltz ?

Reply:
We should always have nuget.config. On my dev machine, I have all 'magic' nuget feeds turned off. This way I know exactly where I am getting packages from.